### ISSUE TITLE
.Net Core Asks me to Install .Net Core SDK

### ISSUE THREAD

**Issue by DanJ210** [Microsoft/vscode-docker#79]
So I created an empty web application from vs2017. Then opened that in VS Code and added docker files which auto-generated, I built the image from the extension and then tried running the image from the extension...

It says, `Did you mean to run dotnet SDK commands? Please install dotnet SDK from: [URL]`

I have dotnet SDK installed already as I can create from the command line no problem. Been installed for a long time. So what am I doing wrong here? Shouldn't this be handled in the image that's downloaded? The one that's auto-generated in the docker file?

**Comment by chrisdias** [comment]
@DanJ210 do you have a sample repo that i can pull your project from?

**Comment by DanJ210** [comment]
@chrisdias Hey I think what was happening is that I was using a production image for a development project, I had to publish the project and use the created .dll. I also realized that there were images for development that had the dotnet SDK installed as well. So I was able to get things to run which were of my own mistakes of "not understanding."

This has all been a learning process for me and I have been able to run projects now. So this original post was kind of silly.

I would have shared the repo but it has sensitive company data so I wouldn't have been able to. I appreciate the services and help that you all provide here. Thank you.
